src/config/Options.js

```diff
@@ -970,6 +970,11 @@ export default class Options {
 			 * @property {Boolean} [legend.show=true] Show or hide legend.
 			 * @property {Boolean} [legend.hide=false] Hide legend
 			 *  If true given, all legend will be hidden. If string or array given, only the legend that has the id will be hidden.
+			 * @property {String|HTMLElement} [legend.contents.bindto=undefined] Template element to bind to.
```

Comment:
This option is convenient, but can consider other way that provide additional css select named "bb-legend" to legend wrapper element like 'title' option. Text to 'title.text' is rendered in `text.bb-title` element and user can customize using css style.

Reply:
I don't think is needed to add more options, neither additional class selector.

`bindto` option accepts css selector or HTMLElement reference, so if user want use `bb-legend` class, just simply can define an element whith that class.

```html
<ul class="bb-legend"></ul>
```

```js
legend: {
	contents: {
		bindto: ".bb-legend"
	}
}
```

For legend title, also can be handled by user defining the structure as user wants.
For ex. if want separate title to be styled with css, them template can be set as follow.

```js
legend: {
	contents: {
		bindto: ".bb-legend",
		template: "<li style=background-color:{=COLOR}><span class=title>{=TITLE}</span></li>"
	}
	...
```

And the result will be as:

```html
<ul class="bb-legend">
	<li style="background-color: rgb(31, 119, 180); visibility: visible; cursor: pointer;" class="bb-legend-item bb-legend-item-a"><span class="title">a</span></li>
	...	
</ul>
```

Giving that result, can be styled using `.bb-legend .title` selector.	

I'll modify the legend title template syntax from `{=NAME}` to `{=TITLE}`, to make more clear and update API doc description.